docs/releases/2.0/dcat-ap-ch.html

```diff
@@ -3974,7 +3978,7 @@ <h2>Property: download URL</h2>
         <tr>
           <th>Usage</th>
           <td><p>
-            Because the distribution mights be callable in different ways (HTTP, WebDAV, FTP),
+            Because the distribution mights be callable in different forms (HTTP, WebDAV, FTP),
```

Comment:
```suggestion
            Because the distribution might be accessible via various methods (HTTP, WebDAV, FTP),
```
'in different forms' makes it sound like different formats are available, but you mean that the same format is available via different download methods, right?